__tests__/get-dependency-list.js

```diff
@@ -12,6 +12,10 @@ const serverless = {
   }
 };
 
+function convertSlashes(paths) {
```

Comment:
Testing on Windows, I was getting issues with the slashes in paths. Figured it was easiest to convert them all.